extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java

```diff
@@ -120,4 +120,16 @@ public static final FlywayDataSourceRuntimeConfig defaultConfig() {
     @ConfigItem(defaultValue = "true")
     public boolean createSchemas;
 
+    /**
+     * Prefix of every placeholder (default: ${ )
+     */
+    @ConfigItem
+    public Optional<String> placeholderPrefix = Optional.empty();
```

Comment:
No need to initialize them

Reply:
But other options are also initializing, so I guess that's fine for this PR

Reply:
Yes, I used the same style of other properties

Reply:
In fact, we might need to initialize them. At least we had to do it for other multi-datasources things, not sure about this particular extension.
The reason is that if you don't have anything set for a given named datasource, the runtime config is not defined at all and we need to be able to instantiate an empty runtime config easily thus everything initialized to the default value.